Server/Plugins/APIDump/main_APIDump.lua

```diff
@@ -1635,6 +1635,73 @@ end
 
 
 
+local function DumpLuaCheck(a_API)
+	LOG("Creating file .luacheckrc...")
+	local file = io.open(".luacheckrc", "w")
+	
+	local Content = [[
+-- Ignore unused function and loop arguments
+unused_args = false
+
+-- Allow self defined globals
+allow_defined = true
+
+-- Ignore this functions
+ignore =
+{
+	"Initialize",  -- Plugin
+	"OnDisable",  -- Plugin
```

Comment:
I think because that the hook / command handler is used in the register function, luacheck doesn't report any problems with them.